ide/utils/shapes.py

```diff
@@ -110,6 +110,50 @@ def repeat(layer):
     return shape
 
 
+def handle_concat_layer(outputId, inputId, net):
```

Comment:
Can't we have a function similar to other layers named `concat` and handle it inside `get_layer_shape`? Do we really need to handle it separately?

Reply:
This is the case for adding the num_output of all the input layers of concat layer, it is needed in topological sort part of shapes.py, where it was being used two times that's why to avoid redundancy I made a separate function

Reply:
Okay.